<launch>
  <node pkg="cr_pkg" type="sensor_fusion.py" name="sensor_fusion" output="screen" />
  <node pkg="cr_pkg" type="mapping_with_known_poses.py" name="mapping_with_known_poses" output="screen"/> 
</launch>
